scripts/pack-debug.sh

```
@@ -0,0 +1,44 @@
+pack_dist() {
+  rm -rf ./tmp/*.zip && mkdir -p ./tmp/
+
+  local git_committish=$(git log --format="%h" -n 1)
+  local target_file=$project_dir/tmp/Rabby_v${app_ver}_debug.${git_committish}.zip
+  local git_utc0_time_linux=$(TZ=UTC0 git show --quiet --date='format-local:%Y-%m-%dT%H:%M:%S+00:00' --format="%cd")
+
+  node $script_dir/fns.js $project_dir/dist $target_file $git_utc0_time_linux;
+
+  get_md5 $target_file;
+  echo ""
+  echo "[pack] (md5: $target_file) $last_md5_value";
```

Shellcheck (0.10.0)

[warning] last_md5_value is referenced but not assigned.

(SC2154)